clang/include/clang/Parse/Parser.h

```diff
@@ -229,6 +230,9 @@ class Parser : public CodeCompletionHandler {
   /// Parsing OpenMP directive mode.
   bool OpenMPDirectiveParsing = false;
 
+  /// Parsing OpenACC directive mode.
+  bool OpenACCDirectiveParsing = false;
```

Comment:
It is unused, need to remove it for now.

Reply:
Its currently being used here: https://github.com/llvm/llvm-project/pull/70234/files#diff-5c51bae1966e0b475927bfa75751eb83db5d90f828aa825cb24f40dea653c408R322 (Line 322 in Parser.cpp).

Reply:
I mean, it has value false only and is not changed anywhere

Reply:
Ah, I see.  As it is likely to be used within another patch or two, I see value in leaving it (if only to not have to edit hte Parser.cpp lines again), but can remove it if you insist.

Reply:
I think better to introduce it with the patch that actually relies on it, currently better to remove it.